style.css

```diff
@@ -0,0 +1,156 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1;
+  letter-spacing: 1.2;
+}
+
+a,
+a:hover,
+a:focus,
+a:active {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  background-color: #e5e5e5;
+  font-size: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.head-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* flex-direction: column; */
+  padding: 40px;
+  background-repeat: no-repeat;
+  background-position: top 80px right 0;
+  border-bottom-left-radius: 80px;
+  background-size: cover;
+}
+
+.white {
+  background-image: url(./images/background-white.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  height: 100vh;
+}
+
+nav {
+  display: flex;
+
+  /* margin-top: 10px; */
+  padding: 10px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+ul {
+  display: flex;
+  gap: 30px;
+  list-style: none;
+}
+
+#list-items {
+  display: none;
+}
+
+li {
+  font-style: normal;
+  font-weight: 500;
+  color: #344563;
+  text-decoration: none;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.bio {
+  /* margin: 0px 0; */
+  background-image: url(./images/mobile-background-img.png);
+  background-size: 100% 95%;
+  background-repeat: no-repeat;
+  padding: 10px;
+  padding-bottom: 118px;
+  border-bottom-left-radius: 15rem;
+}
```

Comment:
Great job making the required changes 👍 . As requested by the previous reviewers, your project design is not in line with the Figma design.

I think you should consider using this styling...

 ```
 width: 100%;
  height: calc(718px - 60px);
 background-image: url(./images/mobile-background-img.png);
  background-position: center;
  background-size: contain;
  background-repeat: no-repeat;
  overflow: hidden;
  border-bottom-left-radius: 5rem;
  background-color: #fff;
  position: relative;
```